### PR TITLE
Use pureconfig for invoker/scheduler's basic http auth

### PR DESCRIPTION
**Pull request by jiangpengcheng**
Replace hardcode username:password

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [x] Scheduler
- [x] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/openwhisk/blob/master/CONTRIBUTING.md#coding-standards) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Review by style95 — APPROVED** [review]
LGTM!

File: core/invoker/src/main/resources/application.conf [hunk 1]
```diff
@@ -176,6 +176,8 @@ whisk {
   }
 
   invoker {
+    username: "invoker.user"
```
Comment (bdoyle0182) [inline core/invoker/src/main/resources/application.conf:179]: Should there be an equivalent default config for the scheduler?

Also don't know off the top of my head but is `.` a special character for curl requests? Wouldn't want to make it harder to curl the invoker with a special character for those that don't care about creds and just using the default

Comment (jiangpengcheng) [reply core/invoker/src/main/resources/application.conf:179]: scheduler already has such config https://github.com/apache/openwhisk/blob/master/core/scheduler/src/main/resources/application.conf#L62-L63

and `.` is not special for curl:

```
[raja@localhost ansible]$ curl -u invoker.user:invoker.pass https://localhost:12001/isEnabled -k
{"enabled":true}
```

**Review by ningyougang — APPROVED** [review]

**Comment by ningyougang** [comment]
LGTM

**Pull request merged**